layouts/layouts/src/androidTest/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutDualScreenTestForSurfaceDuo.kt

```diff
@@ -49,39 +51,31 @@ class FoldableLayoutDualScreenTestForSurfaceDuo {
     }
 
     @Test
+    @SingleScreenTest
     fun testLayoutSingleScreen() {
         onView(withId(R.id.textViewSingle)).check(matches(isDisplayed()))
         onView(withId(R.id.textViewSingle)).check(matches(withText(R.string.single_screen_mode)))
     }
 
     @Test
+    @SingleScreenTest(orientation = UiAutomation.ROTATION_FREEZE_270)
     fun testLayoutSingleScreenLandscape() {
-        windowLayoutInfoConsumer.resetWindowInfoLayoutCounter()
-        uiDevice.setOrientationRight()
-        windowLayoutInfoConsumer.waitForWindowInfoLayoutChanges()
-
         onView(withId(R.id.textViewSingle)).check(matches(isDisplayed()))
         onView(withId(R.id.textViewSingle)).check(matches(withText(R.string.single_screen_mode)))
     }
 
     @Test
+    @DualScreenTest
+    @TargetDevice(device = DeviceModel.SurfaceDuo)
     fun testLayoutDualScreenLandscape() {
-        windowLayoutInfoConsumer.resetWindowInfoLayoutCounter()
-        uiDevice.spanFromStart()
-        windowLayoutInfoConsumer.waitForWindowInfoLayoutChanges()
-
         onView(withId(R.id.textViewDual)).check(matches(isDisplayed()))
         onView(withId(R.id.textViewDual)).check(matches(withText(R.string.dual_portrait)))
     }
 
     @Test
+    @DualScreenTest(orientation = UiAutomation.ROTATION_FREEZE_270)
+    @TargetDevice(device = DeviceModel.SurfaceDuo)
     fun testLayoutDualScreenPortrait() {
```

Comment:
this is unrelated to the test annotations but I think the test names here don't match the strings that are being checked

Reply:
Yeah. I was confused when I see it. But Surface Duo uses resources differently besides regular devices. For example, In portrait, our SurfaceDuo will use resources from res/land folder.

Reply:
Yeah. I was confused when I see it. But Surface Duo uses resources differently besides regular devices. For example, In portrait, our SurfaceDuo will use resources from res/land folder.

Reply:
whoops, thank you for the clarification 😅